nix/manual-overrides.nix

```diff
@@ -42,6 +42,11 @@ hself: hsuper: {
 
   http2-manager = hlib.enableCabalFlag hsuper.http2-manager "-f-test-trailing-dot";
 
+  crypton-connection = hlib.markUnbroken hsuper.crypton-connection;
+  # Patched dependency on crypton-connection
+  HaskellNet-SSL = hlib.markUnbroken hsuper.HaskellNet-SSL;
```

Comment:
if we override this dependency anyways, I don't think we need to mark it unbroken